guides/source/action_mailer_basics.md

```diff
@@ -468,181 +447,220 @@ class UserMailer < ApplicationMailer
+  def welcome_email
+    mail(to: params[:user].email) do |format|
+      format.html { render layout: 'my_layout' }
+      format.text
+    end
+  end
+end
 ```
 
-### Adding Images in Action Mailer Views
+The above will render the HTML part using the `my_layout.html.erb` file and the text part with the usual `user_mailer.text.erb` file.
 
-Unlike controllers, the mailer instance doesn't have any context about the
-incoming request so you'll need to provide the `:asset_host` parameter yourself.
+[`layout`]: https://api.rubyonrails.org/classes/ActionView/Layouts/ClassMethods.html#method-i-layout
 
-As the `:asset_host` usually is consistent across the application you can
-configure it globally in `config/application.rb`:
+Sending Email
+-------------
+
+### Sending Email to Multiple Recipients
+
+It is possible to send an email more than one recipients by setting the `:to`
+field to a list of emails addresses. The list of emails can be an array or a
+single string with the addresses separated by commas.
+
+For example, to inform all admins of a new registration:
 
 ```ruby
-config.action_mailer.asset_host = 'http://example.com'
+class AdminMailer < ApplicationMailer
+  default to: -> { Admin.pluck(:email) },
+          from: 'notification@example.com'
+
+  def new_registration(user)
+    @user = user
+    mail(subject: "New User Signup: #{@user.email}")
+  end
+end
 ```
 
-NOTE: Because we can't infer the protocol from the request, you'll need to
-specify a protocol such as `http://` or `https://` in the
-`:asset_host` config.
+The same format can be used to add more multiple carbon copy (cc) and blind
+carbon copy (bcc) recipients, by setting the `:cc` and `:bcc` keys respectively.
 
-Now you can display an image inside your email.
+### Sending Email with Name
 
-```html+erb
-<%= image_tag 'image.jpg' %>
+It's possible to show the name, instead of the email address, of the person who
+receives the email or sends the email.
+
+To show the name of the person when they receive the email, you can use
+[`email_address_with_name`][] method in `to:`:
+
+```ruby
+def welcome_email
+  @user = params[:user]
+  mail(
+    to: email_address_with_name(@user.email, @user.name),
+    subject: 'Welcome to My Awesome Site'
+  )
+end
+```
+
+The same method in `from:` works to display the name of the sender:
+
+```ruby
+class UserMailer < ApplicationMailer
+  default from: email_address_with_name('notification@example.com', 'Example Company Notifications')
+end
 ```
 
+If the name is a blank string, it returns the email address.
+
+[`email_address_with_name`]: https://api.rubyonrails.org/classes/ActionMailer/Base.html#method-i-email_address_with_name
+
+### Sending Email with Subject Translation
+
+If you don't pass a subject to the mail method, Action Mailer will try to find it in your translations. See the [Internationalization Guide](i18n.html#translations-for-action-mailer-e-mail-subjects) for more.
+
 ### Sending Multipart Emails
```

Comment:
Does this section add much when we have a main section dedicated to multipart emails starting on line 294?